src/components/Header/Header.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import Icon from "images/dns.png";
+import "./style.scss";
+
+class Header extends React.Component {
```

Comment:
choose one way to import "Component". Either you use React or you don't. And then you import differently everywhere